delta/utils/loss/loss_utils_test.py

```diff
@@ -183,6 +183,14 @@ def gen_fake_data(batch_size, embedding_size, num_spks):
                                        limit_to_pi=True)
       self.assertAllClose(output.eval(), output_true)
 
+  def test_focal_loss(self):
+    print('test_focal_loss')
+    with self.cached_session():
+      loss = loss_utils.focal_loss(
+          logits=tf.constant(self.logits),
+          labels=tf.constant(self.labels))
+      self.assertAllClose(loss.eval(), 0.0, rtol=1e-06, atol=1e-6)
```

Comment:
For unit test, I think that it is better to add a test to show the difference between the focal loss and CE, e.g., easy examples get lower losses. Specifically, you may compute the **non-zero loss** to show the correctness, similar to #41  
Thanks!